README.md

```diff
@@ -1,25 +1,78 @@
-# User Requirements for Github Repository Investment Project
-To assess investment potential of all Github repositories, the client would like to be able to identify fast growing repositories as well as analyze a specific repository.
+# Overview
+This project aims to assess the investment potential of all Github repositories. The goal is to be able to identify fast growing repositories as well as analyze a specific repository. The data will be delivered in the format of a SQL database which will update at a daily frequency.
```

Comment:
Looking great now that it's updated, great job! 🙌 

Small nit-pick: Consider using the present "the data _is_ delivered"